### ISSUE TITLE
BaseElements.plugin doesn't load in FileMaker Server 2024 on Sequoia 15.0.1

### ISSUE THREAD

**Issue by sl-lebrit** [GoyaPtyLtd/BaseElements-Plugin#230]
**Describe the bug**
It simply won't load. I've also tried the latest beta: 5.1.0b1 but that doesn't work too; the 4.2.6 version works instead.

**To Reproduce**
We need to know : nothing relevant

The plugin is:
- copied into the proper server directory
- its privileges are set correctly
- the plugin is toggled on in Connectors > Plugins
- FileMaker Server is stopped then restarted

**Expected behavior**
I expect it to be loaded, if I can mix version 4.2.6 on Server and vs. 5.0 on desktop, it's ok...

**Desktop (please complete the following information):**
 - OS and version : macOS Sequoia 15.0.1
 - FileMaker Version : FileMaker Server 21.0.2.202

**Additional context**
- nothing is recorded in the FMS log files; FMS only logs when the plugin is loaded :(

**Comment by nickorr** [comment]
Just confirming you're testing with the 5.1.0b1 build from here :

https://goya.com.au/files/beplugin/5.1.0b1/BaseElements.fmplugin.dmg

Which has a creation date around the 17th of August.  Correct?

I haven't yet run any testing on Sequoia, so can setup and test if needed.

**Comment by sl-lebrit** [comment]
Exactly!Sorry for adding a little extra workThank YouStefanoOn 11 Oct 2024, at 00:20, Nicholas Orr ***@***.***> wrote:﻿
Just confirming you're testing with the 5.1.0b1 build from here :
https://goya.com.au/files/beplugin/5.1.0b1/BaseElements.fmplugin.dmg
Which has a creation date around the 17th of August.  Correct?
I haven't yet run any testing on Sequoia, so can setup and test if needed.

**Comment by nickorr** [comment]
Is this intel or Mac silicon?

**Comment by sl-lebrit** [comment]
Intel!StefanoOn 11 Oct 2024, at 00:48, Nicholas Orr ***@***.***> wrote:﻿
Is this intel or Mac silicon?

**Comment by nickorr** [comment]
Can you test out the new version 5.1.0b3 ( 5.1.0.6 ) and report any issues here : 

https://github.com/GoyaPtyLtd/BaseElements-Plugin/blob/main/docs/Downloads.md
